### ISSUE TITLE
No warning message to enable bluetooth if it was off while the app was in background

### ISSUE THREAD

**Issue by isabelrios** [mozilla-magnet/magnet-client#192]
If the Bluetooth is turned off while the app is in background, when launching it there is no warning to let user know he has to enable bluetooth. The app shows the latest status (tiles if were found) or the Nothing found screen. When forcing refresh it shows Scanning and the Nothing found screen

**Comment by arcturus** [comment]
I think we had this before too right?

**Comment by isabelrios** [comment]
Tried with an older build (Jun 11th commit df335e51) and the message appears if you try to pull down to refresh when bluetooth is off.

**Comment by wilsonpage** [comment]
@isabelrios is this fixed with new mvp?

**Comment by isabelrios** [comment]
@wilsonpage, no, for iOS we see the described behaviour here. If you disable bt while the app is in background, then you go to the app and although you force the pull down to refresh the warning to enable bluetooth is not shown.
